ambry-store/src/main/java/com.github.ambry.store/LogSegment.java

```diff
@@ -55,6 +57,8 @@
   private final AtomicLong endOffset;
   private final AtomicLong refCount = new AtomicLong(0);
   private final AtomicBoolean open = new AtomicBoolean(true);
+  private final Logger logger = LoggerFactory.getLogger(getClass());
```

Comment:
We use static logger in most places - more efficient since one per class rather than instance.

Reply:
good point, will fix this.